src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -222,6 +224,22 @@ public function setColumnWidths(array $widths)
         return $this;
     }
 
+    /**
+     * Sets the maximum width of a column.
```

Comment:
I would expand a bit this description to explain what happens when the contents are longer than the column width: is the text truncated? is it wrapped into multiple lines?

Reply:
added long description :) see below.